### ISSUE TITLE
Improve Grafana Dashboard

### ISSUE THREAD

**Issue by Alex6323** [iotaledger/inx-chronicle#785]

**Comment by Alexandcoats** [comment]
I don't really think we need to do point 1. Firstly, logarithmic axis are not the solution to that problem. Secondly, it's not actually important to know the values, just the trajectory, for that chart. We may want to instead add a separate value indicator that just shows the current size of each table.

**Comment by Alex6323** [comment]
1. I think logarithmic scales solve the problem of data series that play in different ranges of magnitude very well. It's clear that the blocks collection will have orders of magnitude more documents than other collections. 
2. What IMO is important is the size range of a particular collection (so that you get a rough idea). This is solved the logarithmic scale. On the hand... if you have one collection that is 3 orders of magnitude bigger than another, you can't see any trajectory for the smaller one.

**Comment by Alexandcoats** [comment]
Yeah log scales are not the solution. We should have different graphs for each table if they are that dramatically different.

**Comment by grtlr** [comment]
Closing this as non-actionable for now, feel free to reopen at any time.
